USERS.md

```diff
@@ -9,7 +9,8 @@ At Facebook, we use RocksDB as storage engines in multiple data management servi
 4. Laser -- Laser is a high query throughput, low (millisecond) latency, key-value storage service built on top of RocksDB.[1]
 4. Dragon -- a distributed graph query engine. https://code.facebook.com/posts/1737605303120405/dragon-a-distributed-graph-query-engine/
 5. Stylus -- a low-level stream processing framework writtenin C++.[1]
-6. LogDevice -- a distributed data store for logs [2]
+https://en.wikipedia.org/wiki/Stylus_(computing)
```

Comment:
Stylus description can be fit in one line.

Reply:
I undo the changes in Stylus(Description of Stylus is in a single line). assign me this issue so that I can work on it dedicatedly.